File: 02_EstructurasControl/Secuenciales/Python/convertir_temp.py
# Convierte entre Celsius y Fahrenheit
modo = input("C2F o F2C: ").strip().upper()
t = float(input("Temperatura: "))
if modo == "C2F":
    print((t * 9/5) + 32)
elif modo == "F2C":
    print((t - 32) * 5/9)
else:
    print("Modo inválido")

File: 02_EstructurasControl/Secuenciales/CSharp/convertir_temp.cs
using System;
class Program {
  static void Main() {
    string modo = Console.ReadLine().Trim().ToUpper(); // C2F o F2C
    double t = double.Parse(Console.ReadLine());
    if (modo == "C2F") Console.WriteLine((t * 9.0/5.0) + 32.0);
    else if (modo == "F2C") Console.WriteLine((t - 32.0) * 5.0/9.0);
    else Console.WriteLine("Modo inválido");
  }
}
